### ISSUE TITLE
Display warning on screen when shooting someone who is in textmode

### ISSUE THREAD

**Issue by MoonPadUSer** [blue-nebula/base#52]
Display a warning on the screen when you're shooting someone that is in textmode, something like "This person inside the menu, shooting now is a dick move!" or smth along those lines

**Comment by TheAssassin** [comment]
That feature already exists, in this case the chat bubble is shown.

**Comment by MoonPadUSer** [comment]
I know that, the problem is that many newbies don't seem to realise what it's for, maybe my approach is the wrong one. But we need some way to communicate to newbies that they shouldn't shoot people that have this symbol on top of their head

**Comment by TheAssassin** [comment]
> But we need some way to communicate to newbies that they shouldn't shoot people that have this symbol on top of their head

This is more of a docs than a code issue, IMO. But if you really want to you might merge the idea into #35, and maybe display something like a (flashing?) red cross or so?

In any case, you might want to get some feedback in the forums.

**Comment by MoonPadUSer** [comment]
Yeah, I'll make a post in the forum but the idea with merging it with #35 isn't that bad
